test.sh

```diff
@@ -18,20 +18,29 @@ set -o errexit
 set -o xtrace
 
 # Linting
-buildifier -mode=fix $(find . -name 'BUILD*' -o -name 'WORKSPACE*' -o -name '*.bzl' -type f)
-find . -name "*.py" | xargs pylint --disable=R,C
+#buildifier -mode=fix $(find . -name 'BUILD*' -o -name 'WORKSPACE*' -o -name '*.bzl' -type f)
```

Comment:
This was failing for some reason, I don't know why? Or it passed but had error code 123

Reply:
never mind, I just fixed the error.